projects/canopy/src/lib/carousel/carousel.component.ts

```diff
@@ -41,6 +41,14 @@ export class LgCarouselComponent implements AfterContentInit, OnDestroy {
   pause = new BehaviorSubject<boolean>(false);
   pausableTimer$: Observable<void>;
 
+  pauseCarousel() {
```

Comment:
Could you please add a return type of `void` to these two new functions?

Reply:
sure